app/Http/Middleware/Password.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Support\Facades\Auth; 
+class Password
```

Comment:
Rename middleware class name. 
It's should be meaningful according to  it's functions.